### ISSUE TITLE
Deploy AS not appearing in Jupyter Notebooks after Downloading 

### ISSUE THREAD

**Issue by AlexandriaRansom** [dexplo/jupyter_to_medium#42]
Upon installing Jupyter  to Medium ,

after using in command prompt: pip install jupyter_to_medium

Additionally using jupyter bundlerextension enable --py jupyter_to_medium._bundler --sys-prefix

Both installed were successful!

The "Deploy As Medium feature"  was not appearing in Jupyter when pressing the file button . I’ve shut down and restarted the kernel. What would be your recommendations to help resolve this issue to ensure that the deploy as option to medium appears?

**Comment by mjam03** [comment]
Hey,

Sorry for the slow reply - have you started using `jupyter notebook` or are you using `jupyterlab`. Currently the extension only works for jupyter notebook as jupyterlab requires a javascript extension which this is currently not.

Let me know and if `notebook` then please let me know some details about environment / OS etc and I'll try to replicate. So far for me as long as:

`jupyter bundlerextension enable --py jupyter_to_medium._bundler --sys-prefix`

is run then the 'Deploy As' option shows up for me. Also if you run:

`jupyter bundlerextension list`

what does that show?

**Comment by AlexandriaRansom** [comment]
Hi mjam03, 

 Thank you so much for getting back to me! Not a problem. I just tried exactly what you said and opened specifically  **Jupyter** **notebook** in the command prompt to ensure _jupyter lab_ wasn't running. I then opened up a second command prompt window ran _jupyter bundlerextension enable --py jupyter_to_medium._bundler --sys-prefix_ like you mentioned and the **deploy as to medium popped up!** It works perfectly now.

 After, I ran _jupyter bundlerextension list_ and the last line result says "Medium Post" from jupyter_to_medium._bundler enabled .  (Details about the environment Microsoft Windows version 10 Pro). So, I think that was the problem, opening  jupyter lab instead of notebook.  Much appreciated for your help. 

Best,

Alexandria
